bundle/manifests/kuadrant.io_ratelimitpolicies.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   creationTimestamp: null
   labels:
     app: kuadrant
+    gateway.networking.k8s.io/policy: inherited
```

Comment:
`inherited`? Shouldn't be `direct` ?

Reply:
Of course! My distraction! Fixing it now.

Reply:
In fact, I've just noticed that the one in the AuthPolicy is also wrong!!! https://github.com/Kuadrant/kuadrant-operator/blob/3136ed961c704275338aa83ee6c9490efbd5590f/api/v1beta2/authpolicy_types.go#L202

Reply:
Fixed both, and rebased.